core/execution/common/interfaces.go

```diff
@@ -174,31 +204,33 @@ type LiquidityEngine interface {
 	AmendLiquidityProvision(context.Context, *types.LiquidityProvisionAmendment, string) error
 	CancelLiquidityProvision(context.Context, string) error
 	ValidateLiquidityProvisionAmendment(*types.LiquidityProvisionAmendment) error
-	StopLiquidityProvision(context.Context, string)
+	StopLiquidityProvision(context.Context, string) error
 	IsLiquidityProvider(string) bool
 	ProvisionsPerParty() liquidity.ProvisionsPerParty
 	LiquidityProvisionByPartyID(string) *types.LiquidityProvision
 	CalculateSuppliedStake() *num.Uint
 	CalculateSuppliedStakeWithoutPending() *num.Uint
 	UpdatePartyCommitment(string, *num.Uint) (*types.LiquidityProvision, error)
 	EndBlock(*num.Uint, *num.Uint, num.Decimal)
-	UpdateMarketConfig(liquidity.RiskModel, PriceMonitor, *types.LiquiditySLAParams)
+	UpdateMarketConfig(liquidity.RiskModel, liquidity.PriceMonitor, *types.LiquiditySLAParams)
 	OnNonPerformanceBondPenaltySlopeUpdate(num.Decimal)
 	OnNonPerformanceBondPenaltyMaxUpdate(num.Decimal)
 	OnMinProbabilityOfTradingLPOrdersUpdate(num.Decimal)
 	OnProbabilityOfTradingTauScalingUpdate(num.Decimal)
 	OnMaximumLiquidityFeeFactorLevelUpdate(num.Decimal)
 	OnStakeToCcyVolumeUpdate(stakeToCcyVolume num.Decimal)
+	IsPoTInitialised() bool
```

Comment:
Is this right? Looks weird, like it should be IsSpotInitialised

Reply:
what? no, it's probability of trading

Reply:
Ahh ok - sorry! Why not spell it out like it is in the ones above it?

Reply:
mostly cause it's long and I'm lazy :)